content/rest-api/filter.md

```diff
@@ -1217,3 +1217,23 @@ Below you will find the operator allowed to filter on this property, as well as
 ```
 /api/rest/v1/categories?search={"code":[{"operator":"IN","value":["category_code1","category_code2"]}]}
 ```
+
+### By root category
```

Comment:
`By root category` is a little confusing for me.
At first glance, I think it means that I'm able to filter on a root category.
Something like `?root=clothes` and receives all the categories of the tree under this root.

Maybe `By root only` ?